### PR TITLE
Use <For /> in Card documentation

### LINKED ISSUE #3746: Use `<For />` in `Card` documentation
### Provide the exact quote of the error or issue
We should use `<For />` component when repeating a component
e.g.
```tsx
<For each={items}>{(item, index) => <Text key={index} fontWeight="bold">{item}</Text>}</For>
```

https://yamada-ui.com/components/data display/card

### Does the docs page already exist? Please link to it.

_No response_

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### PR DESCRIPTION
**Pull request by yu-yaba**
Closes #3746  

## Description
Fixed the repetition part in the `card` documentation to use the `For` component.

## Is this a breaking change (Yes/No):
No

### CODE REVIEW

File: docs/contents/components/data-display/card/index.ja.mdx [hunk 1]
```diff
@@ -40,147 +40,82 @@ import { Card, CardHeader, CardBody, CardFooter } from "@yamada-ui/react"
...
-      <Text>
-        『SLAM
-        DUNK』（スラムダンク）は、バスケットボールを題材にした井上雄彦による日本の漫画作品。主人公の不良少年桜木花道の挑戦と成長を軸にしたバスケットボール漫画。
-      </Text>
-    </CardBody>
-  </Card>
+  <For
+    each={[{ size: "sm" }, { size: "md" }, { size: "normal" }, { size: "lg" }]}
+  >
+    {({ size }, index) => (
```
Comment (illionillion) [inline docs/contents/components/data-display/card/index.ja.mdx:76]:
```suggestion
    each={["sm", "md", "normal", "lg"]}
  >
    {(size, index) => (
```

File: docs/contents/components/data-display/card/index.ja.mdx [hunk 2]
```diff
@@ -40,147 +40,82 @@ import { Card, CardHeader, CardBody, CardFooter } from "@yamada-ui/react"
...
+  <For
+    each={[
+      { variant: "elevated" },
+      { variant: "outline" },
+      { variant: "subtle" },
+      { variant: "solid" },
+    ]}
+  >
+    {(item, index) => (
+      <Card key={index} variant={item.variant}>
```
Comment (illionillion) [inline docs/contents/components/data-display/card/index.ja.mdx:52]:
```suggestion
  <For
    each={[
      "elevated",
      "outline",
      "subtle",
      "solid",
    ]}
  >
    {(variant, index) => (
      <Card key={index} variant={variant}>
```

File: docs/contents/components/data-display/card/index.mdx
Comment (illionillion) [inline docs/contents/components/data-display/card/index.mdx:1]: @yu-yaba 

Please fix it the same way as the Japanese version.

**Review by illionillion — CHANGES_REQUESTED** [review]
@yu-yaba Thank for PR!

It is cleaner to correct as per the review.

**Comment by yu-yaba** [comment]
@illionillion 
Thanks! 
I fixed it. Please review them.

f4ba1e2a0d8d8290727b0a8550f73f9f3cc5d875

**Review by illionillion — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]
Thanks!

**Pull request merged**